egs/aishell/asr/simple_v1/decode.py

```diff
@@ -137,8 +137,9 @@ def main():
     lang_dir = Path('data/lang_nosp')
     symbol_table = k2.SymbolTable.from_file(lang_dir / 'words.txt')
     phone_symbol_table = k2.SymbolTable.from_file(lang_dir / 'phones.txt')
-    ctc_topo = build_ctc_topo(list(phone_symbol_table._id2sym.keys()))
-    ctc_topo = k2.arc_sort(ctc_topo)
+    phone_ids = get_phone_symbols(phone_symbol_table)
```

Comment:
Could you also update *train.py* to exclude disambiguation symbols while
building the model?

https://github.com/k2-fsa/snowfall/blob/add948544da04ac4bb9c14cef0057c6969c1b9fb/egs/librispeech/asr/simple_v1/train.py#L298

https://github.com/k2-fsa/snowfall/blob/add948544da04ac4bb9c14cef0057c6969c1b9fb/egs/aishell/asr/simple_v1/train.py#L300

The correct one should look like the following:
https://github.com/k2-fsa/snowfall/blob/add948544da04ac4bb9c14cef0057c6969c1b9fb/egs/librispeech/asr/simple_v1/mmi_bigram_train.py#L334-L336

Reply:
sure.